packages/rpc-graphql/src/__tests__/account-test.ts

```diff
@@ -656,47 +638,45 @@ describe('account', () => {
                 data: {
                     account: {
                         data: {
-                            parsed: {
-                                info: {
-                                    authorizedVoters: expect.arrayContaining([
-                                        {
-                                            authorizedVoter: {
-                                                address: 'HMU77m6WSL9Xew9YvVCgz1hLuhzamz74eD9avi4XPdr',
-                                            },
-                                            epoch: expect.any(BigInt),
-                                        },
-                                    ]),
-                                    authorizedWithdrawer: {
-                                        address: 'HMU77m6WSL9Xew9YvVCgz1hLuhzamz74eD9avi4XPdr',
-                                    },
-                                    commission: expect.any(Number),
-                                    epochCredits: expect.arrayContaining([
-                                        {
-                                            credits: expect.any(String),
-                                            epoch: expect.any(BigInt),
-                                            previousCredits: expect.any(String),
-                                        },
-                                    ]),
-                                    lastTimestamp: {
-                                        slot: expect.any(BigInt),
-                                        timestamp: expect.any(BigInt),
-                                    },
-                                    node: {
+                            authorizedVoters: expect.arrayContaining([
```

Comment:
We need to talk about whether to make things like this _plural fields_ or _connections_ to prepare for a future where it's possible to paginate through them.

https://relay.dev/graphql/connections.htm

Reply:
Pagination has been discussed for RPC v2, yeah? Probably good to do.